src/SWP/Bundle/ElasticSearchBundle/Controller/Api/AuthorSearchController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher ElasticSearch Bundle.
+ *
+ * Copyright 2017 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2017 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\ElasticSearchBundle\Controller\Api;
+
+use SWP\Bundle\ElasticSearchBundle\Criteria\Criteria;
+use SWP\Component\Common\Response\ResourcesListResponse;
+use SWP\Component\Common\Response\ResponseContext;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+class AuthorSearchController extends Controller
```

Comment:
this will be a part of SF 5 migration work - remove deprecated controller and use injection for services

Reply:
let's keep it as is for now and then do it properly in another PR as this is not related here right now